applications/asset_tracker_v2/overlay-carrier.conf

```diff
@@ -15,9 +15,6 @@ CONFIG_NRF_MODEM_LIB_SYS_INIT=n
 # The library is only certified for LTE-m
```

Comment:
I see that other NB-IoT "restrictions" have been removed, so just wondering whether this comment is still valid?

Reply:
Removed the limitation (and comment).